subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -359,13 +359,12 @@ static void sw_switch(u8_t dir, u8_t phy_curr, u8_t flags_curr, u8_t phy_next,
 	u8_t cc = SW_SWITCH_TIMER_EVTS_COMP(sw_tifs_toggle);
 	u32_t delay;
 
-	HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI_REGISTER_EVT =
-	    HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI_EVT;
-	HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI_REGISTER_TASK =
-	    HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI_TASK(sw_tifs_toggle);
+	nrf_ppi_channel_endpoint_setup(HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI,
+		HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI_EVT,
+		HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI_TASK(sw_tifs_toggle));
 
-	HAL_SW_SWITCH_RADIO_ENABLE_PPI_REGISTER_EVT(ppi) =
-		HAL_SW_SWITCH_RADIO_ENABLE_PPI_EVT(cc);
+	nrf_ppi_event_endpoint_setup(ppi,
```

Comment:
Why not do the same in line 411 and get rid of `HAL_SW_SWITCH_RADIO_ENABLE_PPI_REGISTER_EVT` completely?

Reply:
I just didn't go thru the code meant only for CONFIG_BT_CTLR_PHY_CODED or CONFIG_SOC_NRF52840 (not supported in the simulated board yet).

Reply:
@aescolar do you mind doing that, for consistency?

Reply:
If you don't mind lets leave it for a future PR. I have my hands quite full now, and this code is very tedious to change. I would not like to botch it.

Reply:
OK. I'll merge as-is, this is not a big deal at this point.